meta/documents/user_guide/en/initial_setup.md

```diff
@@ -82,36 +82,40 @@ This procedure is required only if [the `Send all inventory items to Wayfair` se
 1. From the main Plentymarkets page, go to `Item` >> `Edit item`
 
 2. Search for item(s) and open them
+    **Note:** Verify item variation  says **`main variation`** see image below
 
-3. **For each item**, click `Variations` and open them
+    ![item main variation](../../../images/en/item_main_variation.png)
 
-4. **For each Variation**:
 
-    1. On the `Settings` tab, make sure that the `Active` checkbox in the `Availability` Section is checked:
+     **`For each Item`**
 
-        ![variation active](../../../images/en/variation_active_field.png)
+3. On the `Settings` tab, make sure that the `Active` checkbox in the `Availability` Section is checked:
 
-    2. If the [`Send all inventory items to Wayfair?`](settings_guide.md#send-all-inventory-items-to-wayfair) setting is **disabled**, go to the `Availability` tab of the Variation and add "Wayfair" to the list in the `Markets` area:
+    ![variation active](../../../images/en/variation_active_field.png)
 
-        1. Click in the area that says `Click to select markets`.
+4. If the [`Send all inventory items to Wayfair?`](settings_guide.md#send-all-inventory-items-to-wayfair) setting is **disabled**, go to the `Availability` tab of the Item and add "Wayfair" to the list in the `Markets` area:
 
-        2. Scroll down to the `Wayfair` entry
+    1. Click in the area that says `Click to select markets`.
 
-        3. Place a check mark next to `Wayfair`
+    2. Scroll down to the `Wayfair` entry
 
-        4. Click the `+` button ![add button](../../../images/common/button_plus.png) in the `Markets` area.
+    3. Place a check mark next to `Wayfair`
 
-            ![adding wayfair market to variation](../../../images/en/variation_wayfair_market.png)
+    4. Click the `+` button ![add button](../../../images/common/button_plus.png) in the `Markets` area.
 
-        5. Observe that a `Wayfair` row now exists in `Markets`
+        ![adding wayfair market to variation](../../../images/en/variation_wayfair_market.png)
 
-            ![adding wayfair market to variation](../../../images/en/variation_wayfair_market_added.png)
+    5. Observe that a `Wayfair` row now exists in `Markets`
 
+        ![adding wayfair market to variation](../../../images/en/variation_wayfair_market_added.png)
 
-    3. Click the `Save` left of the Variation `ID`, below the `Global` tab (not the higher-up button for the Item):
 
-        ![variation saving](../../../images/common/variation_save.png)
+5. Click the `Save` left of the `Variation ID`, below the `Global` tab (not the higher-up button for the Item):
 
+    ![variation saving](../../../images/common/variation_save.png)
+
+**Note:** All variations of an item will inherit from the  main variation, unless **`Inheritance is deactivated`**
+    ![inheritance status](../../../images/en/alternate_inheritance.png)

```

Comment:
there should be a line between the text and image in order for it to render correctly.